CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 #### 4.0.4 (To be released)
   Bugs
 
+    * Fix #1147: Cluster context was being ignored when loading the Config from a kubeconfig file
```

Comment:
There is no issue number 1147

Reply:
it's the ID of the PR, not the issue (I don't know why I thought they were shared): https://github.com/fabric8io/kubernetes-client/pull/1147

Reply:
But it should be a issue Number, you fix an issue so